src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -174,9 +185,9 @@
     };
 
     /**
-     * The token types that are ignored while checking "while-idiom".
+     * The token types that are ignored loop checking "loop-idiom".
```

Comment:
> `while checking`

This original wording is the correct usage. It is using the word as a ¿conjunction? instead of talking about the Java while keyword.

Reply:
fixed